packages/angular/cli/utilities/config.ts

```diff
@@ -385,30 +381,6 @@ export function migrateLegacyGlobalConfig(): boolean {
   return false;
 }
 
-// Fallback, check for packageManager in config file in v1.* global config.
-function getLegacyPackageManager(): string | null {
```

Comment:
Was this intended to be removed in the other commit? Not that big a deal to be here, just making sure it was intentional.

Reply:
Yeah